src/libs/ReportUtils.js

```diff
@@ -452,15 +452,15 @@ function getDisplayNamesWithTooltips(participants, isMultipleParticipantReport)
  * @returns {String}
  */
 function getReportName(report, personalDetailsForParticipants = {}, policies = {}) {
-    if (lodashGet(report, 'reportNameValuePairs.type') !== 'chat') {
-        return lodashGet(report, 'reportName', '');
-    }
-
     let formattedName;
-    if (isChatRoom(report)) {
+    if (isDefaultRoom(report)) {
         formattedName = `#${report.reportName}`;
     }
 
+    if (isUserCreatedPolicyRoom(report)) {
```

Comment:
This code makes it look like default rooms don't have `#` baked-in to the reportName, while user-created rooms _do_. This seems like an inconsistency we should fix in Auth.

Reply:
Okay 👍

Reply:
https://github.com/Expensify/Expensify/issues/213278